### PR TITLE
[AC-2204] Finalize sprocs that added the Manage permission (1 of 3)

### PR DESCRIPTION
**Pull request by eliykat**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/AC-2204
## 📔 Objective

When we added the `CollectionUser.Manage` and `CollectionGroup.Manage` columns, we updated a number of sprocs by duplicating them and adding a `_V2` suffix. We later dropped the old, non-suffixed sprocs, but the V2 naming remained.

This is a chore PR that duplicates the V2 sprocs back to non-suffixed versions. After 1 release, updates the repository code to use the non-suffixed versions will be performed. After another release, the V2 sprocs will be dropped, completing the EDD cycle.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: src/Infrastructure.Dapper/AdminConsole/Repositories/OrganizationUserRepository.cs [hunk 1]
```diff
@@ -329,7 +329,7 @@ public async Task<Guid> CreateAsync(OrganizationUser obj, IEnumerable<Collection
         using (var connection = new SqlConnection(ConnectionString))
         {
             var results = await connection.ExecuteAsync(
-                $"[{Schema}].[OrganizationUser_CreateWithCollections_V2]",
+                $"[{Schema}].[OrganizationUser_CreateWithCollections]",
```
Comment (rkac-bw) [inline src/Infrastructure.Dapper/AdminConsole/Repositories/OrganizationUserRepository.cs:332]: The BRE team is looking to leverage EDD to be able to deploy database changes independent of code changes. The general pattern is to just drop old versions, but with renaming it here we are tying the database change with the code. @withinfocus Thoughts?

Comment (withinfocus) [reply src/Infrastructure.Dapper/AdminConsole/Repositories/OrganizationUserRepository.cs:332]: Correct with your catch here. While more tedious, one change and deployment would be your proc duplication (both "V1" and "V2" now having the latter's content), then a second deployment would be the reference back to the V1 version, then finally a third being the V2 drop.

**Comment by eliykat** [comment]
OK, thanks both. The C# changes have been reverted; now it's just duplicating the sprocs.

The next PR will make the C# changes, then a third and final PR will drop the old sprocs.

**Review by rkac-bw — DISMISSED** [review]
lgtm

**Comment by eliykat** [comment]
Had to bump dates on the migration script.

**Review by rkac-bw — APPROVED** [review]

**Review by addisonbeck — APPROVED** [review]

**Pull request merged**